support/support_tokens/README.md

```diff
@@ -0,0 +1,6 @@
+This folder contains the public keys for our support PIV tokens.
+
+`authorized_keys` is from [`oxidecomputer/support-piv-tokens@36abeb82`](https://github.com/oxidecomputer/support-piv-tokens/blob/36abeb82aacb3640be2f6c988e49844356cf65a8/authorized_keys).
```

Comment:
Should we note that these are also in omicron? (And maybe also note in omicron that they're repeated here, in case someone is changing them?)

Reply:
Sure, added a comment and opened https://github.com/oxidecomputer/omicron/pull/6494